website/docs.md

```diff
@@ -1890,6 +1890,18 @@ By default, all listeners are **private**, i.e., accessible only from the cloud
 project's internal network. In our example, we declare that the `hello` listener
 is public.
 
+In order to perform rolling update, an API to be used for health check is required.
```

Comment:
Could you rewrite this leading paragraph as follows:

```
All listeners deployed to GKE are configured to be health-checked by GKE
load-balancers on the `/healthz` HTTP path. To make the `hello` application
compatible with these health checks, augment the application code to have
the HTTP server return status `200` on the `/healthz` endpoint:
```

Reply:
done